README.md

```diff
@@ -440,12 +440,88 @@ You can render non-numeric states by providing state_map config. For example thi
       label: Detected
 ```
 
+#### Group multiple cards
+Group multiple cards with `config-template-card` and add styles with `card-mod`.
+
+<img src="https://user-images.githubusercontent.com/19761269/94990324-d0bac100-0598-11eb-88e2-0e7b24f0013f.png" width="400" >
+
+```yaml
+- cards:
+    - decimals: 0
+      entities:
+        - color: '#04E700'
+          entity: sensor.eth0_in
+        - color: '#FFCC00'
+          entity: sensor.eth0_out
+      font_size_header: 12.5
+      hours_to_show: 1
+      icon: 'mdi:ethernet'
+      line_width: 2.5
+      name: Network
+      points_per_hour: 70
+      show:
+        fill: false
+        icon_adaptive_color: true
+        labels: true
+        legend: false
+        points: false
+        state: false
+      style: |
+        ha-card > div:nth-child(-n+2) {
+          padding: 0 14px 0 14px !important
+        }
+        ha-card > .info {
+          padding: 0 14px 0px 14px !important
+        }
+      type: 'custom:mini-graph-card'
+    - cards:
+        - entities:
+            - color: '#04E700'
+              entity: sensor.eth0_in_total
+              name: Traffic In
+          font_size: 78
+          font_size_header: 12.5
+          show:
+            graph: false
+            icon_adaptive_color: true
+            name_adaptive_color: true
+          style: |
+            ha-card > div:nth-child(-n+2) {
+              padding: 0 14px 0 14px !important
+            }
+            ha-card > .info {
+              padding: 0 14px 0px 14px !important
+            }
+          type: 'custom:mini-graph-card'
+        - align_icon: left
+          align_state: right
+          entities:
+            - color: '#FFCC00'
+              entity: sensor.eth0_out_total
+              name: Traffic Out
+          font_size: 78
+          font_size_header: 12.5
+          show:
+            icon_adaptive_color: true
+            name_adaptive_color: true
+            graph: false
+          style: |
+            ha-card > div:nth-child(-n+2) {
+              padding: 0 14px 0 14px !important
+            }
+            ha-card > .info {
+              padding: 0 14px 0px 14px !important
+            }
+          type: 'custom:mini-graph-card'
+      type: horizontal-stack
+  type: 'custom:vertical-stack-in-card'
```

Comment:
Can you please move these to the top of the section? It is hard to read the configuration if the type entries are not on the top - at least for me